addon-api/types.d.ts

```diff
@@ -0,0 +1,25 @@
+import UserscriptAddon from "./content-script/Addon.js";
+import PersistentScriptAddon from "./background/Addon.js";
+
+interface Script {
+  global: Object;
```

Comment:
`Object` means `any`
do `{ [key: string]: string }` to specify objects

Reply:
heh, typescript is weird